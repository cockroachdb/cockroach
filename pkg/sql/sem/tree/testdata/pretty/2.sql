SELECT count(*) count, winner, counter * 60 * 5 as counter FROM (SELECT winner, round(length / 60 / 5) as counter FROM players WHERE build = $1 AND (hero = $2 OR region = $3)) GROUP BY winner, counter
