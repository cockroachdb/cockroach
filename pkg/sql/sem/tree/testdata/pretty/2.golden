1:
-
SELECT
	count(
		*
	)
		AS count,
	winner,
	counter
	* 60
	* 5
		AS counter
FROM
	(
		SELECT
			winner,
			round(
				length
				/ 60
				/ 5
			)
				AS counter
		FROM
			players
		WHERE
			build
			= $1
		AND (
				hero
				= $2
			 OR region
				= $3
			)
	)
GROUP BY
	winner,
	counter

6:
------
SELECT
	count(
		*
	)
		AS count,
	winner,
	counter
	* 60
	* 5
		AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM
			 players
		 WHERE
			 build
			 = $1
		 AND (
				 hero
				 = $2
			  OR region
				 = $3
			 )
	 )
GROUP BY
	winner,
	counter

12:
------------
SELECT
	count(*)
		AS count,
	winner,
	counter
	* 60
	* 5
		AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM
			 players
		 WHERE
			 build
			 = $1
		 AND (
				 hero
				 = $2
			  OR region
				 = $3
			 )
	 )
GROUP BY
	winner,
	counter

13:
-------------
SELECT count(
		   *
	   )
		   AS count,
	   winner,
	   counter
	   * 60
	   * 5
		   AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM
			 players
		 WHERE
			 build
			 = $1
		 AND (
				 hero
				 = $2
			  OR region
				 = $3
			 )
	 )
GROUP BY
	winner,
	counter

15:
---------------
SELECT count(*)
		   AS count,
	   winner,
	   counter
	   * 60
	   * 5
		   AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM
			 players
		 WHERE
			 build
			 = $1
		 AND (
				 hero
				 = $2
			  OR region
				 = $3
			 )
	 )
GROUP BY
	winner,
	counter

16:
----------------
SELECT count(*)
		   AS count,
	   winner,
	   counter
	   * 60
	   * 5
		   AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM
			 players
		 WHERE
			 build
			 = $1
		 AND (
				 hero
				 = $2
			  OR region
				 = $3
			 )
	 )
GROUP BY winner,
		 counter

20:
--------------------
SELECT count(*)
		   AS count,
	   winner,
	   counter
	   * 60
	   * 5
		   AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM
			 players
		 WHERE build
			   = $1
		   AND (
				   hero
				   = $2
				OR region
				   = $3
			   )
	 )
GROUP BY winner,
		 counter

21:
---------------------
SELECT count(*)
		   AS count,
	   winner,
	   counter
	   * 60
	   * 5
		   AS counter
FROM (
		 SELECT
			 winner,
			 round(
				 length
				 / 60
				 / 5
			 )
				 AS counter
		 FROM players
		 WHERE build
			   = $1
		   AND (
				   hero
				   = $2
				OR region
				   = $3
			   )
	 )
GROUP BY winner,
		 counter

23:
-----------------------
SELECT count(*)
		   AS count,
	   winner,
	   counter * 60 * 5
		   AS counter
FROM (
		 SELECT winner,
				round(
					length
					/ 60
					/ 5
				)
					AS counter
		 FROM players
		 WHERE build
			   = $1
		   AND (
				   hero
				   = $2
				OR region
				   = $3
			   )
	 )
GROUP BY winner,
		 counter

24:
------------------------
SELECT count(*)
		   AS count,
	   winner,
	   counter * 60 * 5
		   AS counter
FROM (
		 SELECT winner,
				round(
					length
					/ 60
					/ 5
				)
					AS counter
		 FROM players
		 WHERE build
			   = $1
		   AND (
				   hero
				   = $2
				OR region
				   = $3
			   )
	 )
GROUP BY winner, counter

25:
-------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5
		   AS counter
FROM (
		 SELECT winner,
				round(
					length
					/ 60
					/ 5
				)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (
				   hero
				   = $2
				OR region
				   = $3
			   )
	 )
GROUP BY winner, counter

28:
----------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5
		   AS counter
FROM (
		 SELECT winner,
				round(
					length
					/ 60
					/ 5
				)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (
				   hero = $2
				OR region
				   = $3
			   )
	 )
GROUP BY winner, counter

30:
------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5
		   AS counter
FROM (
		 SELECT winner,
				round(
					length
					/ 60
					/ 5
				)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (
				   hero = $2
				OR region = $3
			   )
	 )
GROUP BY winner, counter

34:
----------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner,
				round(
					length
					/ 60
					/ 5
				)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (
				   hero = $2
				OR region = $3
			   )
	 )
GROUP BY winner, counter

35:
-----------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner,
				round(
					length / 60 / 5
				)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (
				   hero = $2
				OR region = $3
			   )
	 )
GROUP BY winner, counter

38:
--------------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner,
				round(length / 60 / 5)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (
				   hero = $2
				OR region = $3
			   )
	 )
GROUP BY winner, counter

41:
-----------------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner,
				round(length / 60 / 5)
					AS counter
		 FROM players
		 WHERE build = $1
		   AND (hero = $2 OR region = $3)
	 )
GROUP BY winner, counter

49:
-------------------------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner,
				round(length / 60 / 5) AS counter
		 FROM players
		 WHERE build = $1
		   AND (hero = $2 OR region = $3)
	 )
GROUP BY winner, counter

56:
--------------------------------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner,
				round(length / 60 / 5) AS counter
		 FROM players
		 WHERE build = $1 AND (hero = $2 OR region = $3)
	 )
GROUP BY winner, counter

57:
---------------------------------------------------------
SELECT count(*) AS count,
	   winner,
	   counter * 60 * 5 AS counter
FROM (
		 SELECT winner, round(length / 60 / 5) AS counter
		 FROM players
		 WHERE build = $1 AND (hero = $2 OR region = $3)
	 )
GROUP BY winner, counter

61:
-------------------------------------------------------------
SELECT count(*) AS count, winner, counter * 60 * 5 AS counter
FROM (
		 SELECT winner, round(length / 60 / 5) AS counter
		 FROM players
		 WHERE build = $1 AND (hero = $2 OR region = $3)
	 )
GROUP BY winner, counter

116:
--------------------------------------------------------------------------------------------------------------------
SELECT count(*) AS count, winner, counter * 60 * 5 AS counter
FROM (SELECT winner, round(length / 60 / 5) AS counter FROM players WHERE build = $1 AND (hero = $2 OR region = $3))
GROUP BY winner, counter

203:
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SELECT count(*) AS count, winner, counter * 60 * 5 AS counter FROM (SELECT winner, round(length / 60 / 5) AS counter FROM players WHERE build = $1 AND (hero = $2 OR region = $3)) GROUP BY winner, counter
